### ISSUE TITLE
python-pip3 is a required package

### ISSUE THREAD

**Issue by MJoergen** [glaure/tortoisehg4ubuntu#2]
After a clean install of Linux Mint 20.2, I try to install tortoisehg, but get the following error:

```
...
/var/lib/dpkg/tmp.ci/preinst: 7: pip3: not found
dpkg: error processing archive /tmp/apt-dpkg-install-cPIIeQ/5-tortoisehg_5.8.1-g
laure_amd64.deb (--unpack):
 new tortoisehg package pre-installation script subprocess returned error exit s
tatus 127
Errors were encountered while processing:
 /tmp/apt-dpkg-install-cPIIeQ/5-tortoisehg_5.8.1-glaure_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

The solution is to first install the prerequisite package:
```
sudo apt install python3-pip
```

**Comment by glaure** [comment]
You are correct. I will add it as a dependency.

**Comment by glaure** [comment]
Fixed with v5.9.1
